### ISSUE TITLE
A way to identify a sandboxed worker

### ISSUE THREAD

**Issue by ppedziwiatr** [taskforcesh/bullmq#1545]
Hey,

is there a way to somehow name and identify sandboxed workers - assuming that I'm creating them with concurrency set to > 1.

e.g. sth like
Worker-1
Worker-2
...
Worker-8

- mainly for logging purposes.

**Comment by andrisi** [comment]
Yes, this would be great, for any worker. Then on some monitoring interface I could see which workers are actually doing anything, as I'd see more then just an IP:port for each worker as queue.getWorkers() returns.

**Comment by manast** [comment]
We are exploring the idea of storing as job locks some unique worker ID, so that we can map which jobs that are currently active with the current online workers.

**Comment by andrisi** [comment]
FYI: https://github.com/taskforcesh/bullmq/discussions/2681
